attributes/default.rb

```diff
@@ -39,7 +33,8 @@
     snapRetainCount: 1,
     purgeInterval: 1
   },
-  "authProvider.1": "org.apache.zookeeper.server.auth.SASLAuthenticationProvider"
+  "authProvider.1": "org.apache.zookeeper.server.auth.SASLAuthenticationProvider",
+  "admin.enableServer": false
```

Comment:
I'm not sure if we use it though but why disable it? also, will this change at any point should we expose it as an attribute?

Reply:
No we don't use it. The admin server UI was introduced in Zookeeper 3.5. I don't think it will change to be honest. If we were to enable it, we would also need to care about authentication and authorization of the UI. 

Since we have Zookeeper just to support Kafka, I don't think we need the admin web ui.